shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/external/MarshallerTest.java

```diff
@@ -73,6 +85,54 @@ public void typical() throws Exception {
         .contains("    database-name: databaseName")
         .contains("    table-name: tableName")
         .contains("    table-location: replicaTableLocation");
+
+  }
+
+  @Test
+  public void typicalWithGraphite() throws Exception {
+    CircusTrainConfig config = CircusTrainConfig
+        .builder()
+        .sourceName("sourceName")
+        .sourceMetaStoreUri("sourceMetaStoreUri")
+        .replicaName("replicaName")
+        .replicaMetaStoreUri("replicaMetaStoreUri")
+        .copierOption("p1", "val1")
+        .copierOption("p2", "val2")
+        .replication(ReplicationMode.FULL, "databaseName", "tableName", "replicaTableLocation", Arrays.asList("part"),
+            Arrays.asList(Arrays.asList("partval")))
+        .graphite(graphiteConfig)
+        .build();
+
+    File file = tmp.newFile("conif.yml");
+    marshaller.marshall(file.getAbsolutePath(), config);
+    assertThat(Files.readLines(file, StandardCharsets.UTF_8))
+        .contains("source-catalog:")
+        .contains("  disable-snapshots: false")
+        .contains("  name: sourceName")
+        .contains("  hive-metastore-uris: sourceMetaStoreUri")
+        .contains("replica-catalog:")
+        .contains("  name: replicaName")
+        .contains("  hive-metastore-uris: replicaMetaStoreUri")
+        .contains("copier-options:")
+        .contains("  p1: val1")
+        .contains("  p2: val2")
+        .contains("table-replications:")
+        .contains("  replication-mode: FULL")
+        .contains("  source-table:")
+        .contains("    database-name: databaseName")
+        .contains("    table-name: tableName")
+        .contains("    partition-filter: (part='partval')")
+        .contains("    partition-limit: 32767")
+        .contains("  replica-table:")
+        .contains("    database-name: databaseName")
+        .contains("    table-name: tableName")
+        .contains("    table-location: replicaTableLocation")
+        .contains("graphite-config:")
+        .contains("  enabled: false")
```

Comment:
For this test it should be configured to be enabled true

Reply:
Same for the test underneath, right?